pkg/operator/ceph/cluster/osd/config.go

```diff
@@ -91,6 +108,20 @@ func (c *Cluster) osdRunFlagTuningOnPVC(osdID int) error {
 	return nil
 }
 
+func (c *Cluster) osdRunFlagFastTuningOnPVC(osdID int) error {
+	who := fmt.Sprintf("osd.%d", osdID)
+	monStore := opconfig.GetMonStore(c.context, c.clusterInfo)
+
+	for flag, val := range defaultTuneFastSettings {
+		err := monStore.Set(who, flag, val)
```

Comment:
If the command line params don't work out, how about executing `monStore.Set()` in a goroutine? Then all the settings can be applied in parallel.

Reply:
Moved to CLI so, changed.